API.Tests/Parser/MangaParserTests.cs

```diff
@@ -26,6 +26,7 @@ public MangaParserTests(ITestOutputHelper testOutputHelper)
         [InlineData("Akame ga KILL! ZERO v01 (2016) (Digital) (LuCaZ).cbz", "1")]
         [InlineData("v001", "1")]
         [InlineData("Vol 1", "1")]
+        [InlineData("vol_356-1", "356")] // Mangapy syntax
```

Comment:
What is the -1? Is it a chapter, if so, we should add a case for it.

Reply:
-1, -2, -3, etc is a number for corrections, if they find spelling errors and so on they upload a new version and increase that number. In theory, at least, all the manga I get downloaded have -1 though (weird).